src/components/submission/list.vue

```diff
@@ -485,27 +485,44 @@ export default {
 }
 
 #submission-list-actions {
-  align-items: baseline;
+  align-items: center;
   display: flex;
   flex-wrap: wrap-reverse;
+  // This is 10px of space between actions on the row and also 10px between
+  // rows if the actions start wrapping.
+  gap: 10px;
+  margin-bottom: 25px;
+
+  .form-inline {
+    margin-bottom: 0;
+    padding-bottom: 0;
+  }
 }
 #submission-field-dropdown {
+  // This is the entire spacing between the dropdown and the filters to its
+  // left. Since they're both children of the form, the flexbox gap doesn't
+  // apply to them.
   margin-left: 15px;
-  margin-right: 5px;
-}
-#submission-list-refresh-button {
-  margin-left: 10px;
+  // Additional space between the dropdown and the refresh button
   margin-right: 5px;
 }
 #submission-download-button {
-  // The bottom margin is for if the download button wraps above the other
-  // actions.
-  margin-bottom: 10px;
   margin-left: auto;
 }
 
 #submission-list-test-in-browser {
```

Comment:
The presence of this button is how I'm styling things differently on the draft testing page.

Reply:
I've now added a code comment explaining this strategy.